packages/sku/scripts/translations.js

```diff
@@ -32,52 +32,70 @@ const ensureBranch = () => {
   console.log(`Using branch ${branch} for Phrase translations`);
 };
 
+const log = (message) => console.log(chalk.cyan(message));
+
 (async () => {
   const translationSubCommand = commandArguments[0];
 
-  console.log(chalk.cyan('Translations', translationSubCommand));
-
   const vocabConfig = getVocabConfig();
 
   if (!vocabConfig) {
     console.log(
-      'No languages configured. Please set languages in sku.config.js before running translation commands',
+      'No languages configured. Please set languages in  before running translation commands',
     );
   }
 
   if (!translationSubCommands.includes(translationSubCommand)) {
     throw new Error(
-      `Unknown command ${translationSubCommand}. Available options are ${translationSubCommands.join(
+      `Unknown sub-command ${translationSubCommand}. Available options are ${translationSubCommands.join(
         ', ',
-      )}`,
+      )}.`,
     );
   }
 
   try {
     if (translationSubCommand === 'compile') {
-      console.log('Watching for changes to translations');
-      compile({ watch }, vocabConfig);
+      log('Compiling translations...');
+
+      if (watch) {
+        log('Watching for changes to translations');
+        // Don't await so it can run in the background
+        compile({ watch }, vocabConfig);
```

Comment:
Am I correct that not using `await` here will prevent the `catch` block below from catching any exceptions thrown here and thus ensure the process does not exit on error? 

If that is the case it might be better to include this detail in the comment as "run in the background" is a bit vague.

Reply:
https://github.com/seek-oss/sku/pull/1075/commits/a7392fee9c54f1865f7d7ea25588f54b99b8ce6a